src/cascadia/TerminalApp/AppKeyBindings.idl

```diff
@@ -80,29 +15,6 @@ namespace TerminalApp
         Microsoft.Terminal.Settings.KeyChord GetKeyBindingForAction(ShortcutAction action);
         Microsoft.Terminal.Settings.KeyChord GetKeyBindingForActionWithArgs(ActionAndArgs actionAndArgs);
 
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> CopyText;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> PasteText;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> NewTab;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> OpenNewTabDropdown;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> DuplicateTab;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> NewWindow;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> CloseWindow;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> CloseTab;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ClosePane;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> SwitchToTab;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> NextTab;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> PrevTab;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> SplitVertical;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> SplitHorizontal;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> AdjustFontSize;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ResetFontSize;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ScrollUp;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ScrollDown;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ScrollUpPage;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ScrollDownPage;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> OpenSettings;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ResizePane;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> MoveFocus;
-        event Windows.Foundation.TypedEventHandler<AppKeyBindings, ActionEventArgs> ToggleFullscreen;
+        void SetDispatch(ShortcutActionDispatch dispatch);

```

Comment:
Probably should be `ShortcutActionDispatch Dispatch{ set; }`

Reply:
I hate that you're right and I have to spin a build for this, but you're right

Reply:
wait I don't think you can have just a setter: https://docs.microsoft.com/en-us/uwp/midl-3/intro
> The Windows Runtime doesn't support write-only properties. But you can specify only the set keyword to revise an existing read-only property into a read-write property. Take this version of Area as an example.

Reply:
Cha-ching